package.json

```diff
@@ -17,7 +17,7 @@
     "test:lint": "npm-run-all --parallel --aggregate-output test:lint:*",
     "test:lint:js": "eslint .",
     "test:lint:md": "remark -qf .",
-    "test:lint:stylelint": "stylelint \"layouts/css/**/*.{css,scss}\"",
+    "test:lint:stylelint": "stylelint \"layouts/css/**/*.{css,scss}\" --rd",
```

Comment:
What do the new flags do?

Reply:
Reports needless disables. Too lazy to split it, sorry 👼